src/Zicht/Bundle/HtmldevBundle/Resources/views/_base.html.twig

```diff
@@ -3,13 +3,14 @@
     <head>
         <meta charset="UTF-8">
         <meta http-equiv="x-ua-compatible" content="ie=edge">
-        <title>{% block head_title '' %}</title>
+        <title>{%- if app.environment != 'production' %}{{ assets_version() }} - {% endif %}{% block head_title '' %}</title>
```

Comment:
Heel goed!

Vraag me alleen af of het niet beter/mooier is om erachter te zetten. En evt juist in de template te zetten die het `head_title` block defineert/vult?

Reply:
Doorgaans zijn er veel plaatsen waar de head_title gezet wordt, dan is dit consistenter.

En het liefst helemaal vooraan... tijdens testen ben je eigenlijk niet geinteresseerd meer in de titel, als je eenmaal weet dat die klopt, maar de versie blijft elke F5 weer relevant.

Reply:
Ok prima!